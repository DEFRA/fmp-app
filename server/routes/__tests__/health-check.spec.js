const {
  submitGetRequest
} = require('../../__test-helpers__/server')

describe('healthcheck', () => {
  it('Should get /healthcheck successfully', async () => {
    const response = await submitGetRequest({ url: '/healthcheck' })
    expect(response.result).toMatchSnapshot()
  })
  it('Should get /health-check successfully', async () => {
    const response = await submitGetRequest({ url: '/health-check' })
    expect(response.result).toMatchSnapshot()
  })
})
