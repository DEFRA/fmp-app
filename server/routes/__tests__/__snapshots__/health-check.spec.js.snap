// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`cookies Should get /health-check successfully 1`] = `"{"name":"fmp-app","version":"v3.0.0-1","revision":"925617123"}"`;

exports[`cookies Should get /healthcheck successfully 1`] = `"{"name":"fmp-app","version":"v3.0.0-1","revision":"925617123"}"`;
